### ISSUE TITLE
dropdown: menu closes when scrolling inside menu

### ISSUE THREAD

**Issue by jaketrent** [pluralsight/design-system#1741]
Related:
https://pluralsight.slack.com/archives/C70HPSJPP/p1620672334085100

Support:
- Scrolling via mouse/wheel
- Scroll via keyboard -- double-check WCAG recommendations on which keys to support
